operator/Makefile

```diff
@@ -0,0 +1,151 @@
+
```

Comment:
let's clean up this Makefile and only keep the targets we actually use.

Reply:
done